src/modules/StudentsTable/components/Dashboard/components/TableHead/index.tsx

```diff
@@ -5,18 +5,23 @@ import {
   StyledTableHeadRow,
   StyledTableHeader,
 } from './styled';
+import { useTranslation } from 'react-i18next';
 
-export const TableHead: FC = () => (
-  <StyledTableHead>
-    <StyledTableHeadRow>
-      {TABLE_HEADERS.map((tableHeader: string, index: number) => (
-        <StyledTableHeader
-          className={`TableItem--${index}`}
-          key={`TableHeaderKey-${index}`}
-        >
-          {tableHeader}
-        </StyledTableHeader>
-      ))}
-    </StyledTableHeadRow>
-  </StyledTableHead>
-);
+export const TableHead: FC = () => {
+  const { t } = useTranslation();
+
+  return (
+    <StyledTableHead>
+      <StyledTableHeadRow>
+        {TABLE_HEADERS.map((tableHeader: string, index: number) => (
+          <StyledTableHeader
+            className={`TableItem--${index}`}
+            key={`TableHeaderKey-${index}`}
```

Comment:
it's not critical there, because TABLE_HEADERS is constant, but `key={`TableHeaderKey-${index}`}` this key is bad you know why

Reply:
This refacor exists in 0 branch